hudi-cli/src/main/java/org/apache/hudi/cli/commands/FileSystemViewCommand.java

```diff
@@ -239,9 +239,9 @@ private HoodieTableFileSystemView buildFileSystemView(String globRegex, String m
     HoodieTableMetaClient client = HoodieCLI.getTableMetaClient();
     HoodieTableMetaClient metaClient = HoodieTableMetaClient.builder()
         .setConf(client.getStorageConf().newInstance())
-        .setBasePath(client.getBasePath()).setLoadActiveTimelineOnLoad(true).build();
+        .setBasePath(client.getBasePath().toString()).setLoadActiveTimelineOnLoad(true).build();
```

Comment:
nit: can some of the `toString()` calls be avoided by directly passing the `StoragePath` instance?